packages/protocol/test-sol/unit/governance/network/Governance.t.sol

```diff
@@ -3388,6 +3505,59 @@ contract GovernanceTest_resetHotfix is GovernanceTest {
   bytes32 hotfixHash;
   event HotfixRecordReset(bytes32 indexed hash);
 
+  function setUp() public {
+    super.setUp();
+
+    address val1 = actor("validator1");
+    governance.addValidator(val1);
+    vm.prank(val1);
+    accounts.createAccount();
+
+    // _whenL2();
+    vm.prank(accOwner);
+    governance.setSecurityCouncil(accCouncil);
+
+    hotfixHash = governance.getHotfixHash(
+      okProp.values,
+      okProp.destinations,
+      okProp.data,
+      okProp.dataLengths,
+      SALT
+    );
+  }
+
+  function test_Reverts_whenCalledOnL1() public {
+    vm.prank(accOwner);
+    governance.setHotfixExecutionTimeWindow(DAY);
+
+    vm.prank(accApprover);
+    governance.approveHotfix(HOTFIX_HASH);
+
+    (bool approved, , ) = governance.getHotfixRecord(HOTFIX_HASH);
+
+    assertTrue(approved);
+
+    vm.roll(block.number + governance.getEpochSize());
+    vm.prank(actor("validator1"));
```

Comment:
`validator1` should be a storage variable and used here, rather than calling `actor` again.